aa-sdk/core/src/utils/traceHeader.ts

```diff
@@ -1,10 +1,26 @@
 function generateRandomHexString(numBytes: number) {
-  const bytes = crypto.getRandomValues(new Uint8Array(numBytes));
-  const array = Array.from(bytes);
-  const hexPairs = array.map((b) => b.toString(16).padStart(2, "0"));
+  const hexPairs = new Array(numBytes).fill(0).map(() =>
+    Math.floor(Math.random() * 16)
+      .toString(16)
+      .padStart(2, "0")
+  );
```

Comment:
Thank you @jakehobbs and @moldy530 for helping to solve the random issue with crypto. Since this doesn't have to be secure, we are going with random. We just want a random distrubution